setup.py

```diff
@@ -13,7 +13,7 @@
 
 install_requires=[
         'svreal>=0.2.2',
-        'msdsl>=0.2.5',
+        'msdsl>=0.3.5',
```

Comment:
The ``as_posix()`` fix added to ``msdsl`` hadn't been released to PyPI, so I bumped the version for ``msdsl`` and issued a new release.  In future PRs, I think the ``version`` should be bumped to avoid this type of issue (applies to ``anasymod``, ``msdsl``, and ``svreal``).